dist/gqlRules/gqlOperationName/gqlOperationName.js

```diff
@@ -37,24 +37,29 @@ const meta = {
                 },
```

Comment:
Ignore this file, this is the resulting build file. Changes are in: `src/gqlRules/gqlOperationName/gqlOperationName.js` below.